lib/benchee/formatters/console.ex

```diff
@@ -21,9 +23,9 @@ defmodule Benchee.Formatters.Console do
   """
   @spec output(Suite.t) :: Suite.t
   def output(suite = %Suite{}) do
-    suite
-    |> format
-    |> IO.write
+    _ = suite
```

Comment:
I think here or in the original `write` we should proabably match on `:ok` to see that everything went well?

Reply:
Yeah, error handling is still an open question here. Part of this was still because I was confused about `IO.write/2`, but that's been cleared up now (and I fixed the confusing part of that documentation). It turns out there _isn't_ some undocumented error case, and `IO.write/2` always returns `:ok`, so we can match on that here to make dialyzer happy.